lib/matplotlib/figure.py

```diff
@@ -1253,11 +1253,13 @@ def colorbar(
         # Store the value of gca so that we can set it back later on.
         if cax is None:
             if ax is None:
-                raise ValueError(
+                _api.warn_deprecated(3.6, message=(
```

Comment:
Surprised that worked, but I guess it's flexible; usually we pass a string:
```suggestion
                _api.warn_deprecated('3.6', message=(
```

Reply:
ooops!